app/models/document.rb

```diff
@@ -0,0 +1,3 @@
+class Document < ApplicationRecord
+  belongs_to :vacancy, optional: true
```

Comment:
On hindsight, I think we want the `optional: true` to be on the Vacancy model.
A vacancy could **have no documents attached** but a document **must have a vacancy it is attached to**.